### PR TITLE
feat: support for minimum & maximum track styles

### LINKED ISSUE #354: Different styles for minimum and maximum track
Need props to style minimum and maximum tracks differently.
In my case I need different heights and border radius.

### PR DESCRIPTION
**Pull request by g7i**
User can now pass `minimumTrackStyle` and `maximumTrackStyle`.

🗒️ Note - This PR **doesn't** contain breaking changes.

This PR closes: #354

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -622,6 +622,8 @@ export class Slider extends PureComponent<SliderProps, SliderState> {
             thumbStyle,
             thumbTintColor,
             trackStyle,
+            minimumTrackStyle: propMinimumTrackStyle,
```
Comment (g7i) [inline src/index.tsx:625]: `minimumTrackStyle` has already been defined in the code. Hence renamed the prop to `propMinimumTrackStyle`

**Comment by matpiwowarski** [comment]
this feature would be very helpful!

**Review by miblanchard — APPROVED** [review]
Thank you for this good feature addition 😄

**Comment by Samipya** [comment]
Hi, is this feature available in 2.2.0 ?

**Pull request merged**